docs/public/index.css

```diff
@@ -183,13 +183,14 @@ strong {
 }
 
 /* Supporting Content */
+code {
+  font-family: var(--font-mono);
+}
 
 code:not([class*='language']) {
```

Comment:
Would it make sense to resolve https://github.com/snowpackjs/astro/issues/760 here? Or maybe that should be a different PR especially since Nate hasn't commented on whether that was intentional...

Reply:
Makes sense to me, I'll do it without tackling a larger CSS `code / pre` refactor just yet